### PR TITLE
Update devkit-activation.md

### PR DESCRIPTION
**Pull request by FLY1NGSQU1RR3L**
**Major Edit** 
Added clarification that Retail kits are different from a software perspective than Development Kits (hardware carrying an entitlement only available via our NDA program) and behave differently in Developer Mode.  This article is intended for non-managed development partners, but needs to provide clarification about this delta for developers with access to both Retail and NDA Developer Kits (only Xbox One X, Xbox Series X, and Xbox Series S devkits look physically different than their retail counterparts).

**Minor Edit** 
Updated reference(s) from "Xbox One" console to "Xbox" console where the guidance applies equally to Xbox One and Xbox Series X|S consoles.

### CODE REVIEW

**Comment by PRMerger13** [comment]
@FLY1NGSQU1RR3L : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by FLY1NGSQU1RR3L** [comment]
Testing the process for editing this repo.  If a manual approval is required, please email clintw@microsoft.com so I can document the process for our PMs and writers.

**Review by KevinAsgari — APPROVED** [review]

**Comment by Jak-MS** [comment]
@FLY1NGSQU1RR3L @KevinAsgari - I'll be making a few editorial tweaks so we can merge this.

-----

@FLY1NGSQU1RR3L - Here's some information about making content updates, but I'm also sending you an email:

> Microsoft authors and contributors need to make content contributions through the private repository so the content can be staged and validated by the current validation rules.  (in this case: https://github.com/MicrosoftDocs/windows-uwp-pr ).
> 
> After you review staging and fix all validation issues, you can tag an author for review or add the sign-off comment to let PR reviewers know that the updates are ready to be merged.  
> 
> Read about [authors working in the public repo](https://review.docs.microsoft.com/en-us/help/contribute/contribute-authors-in-public-repo?branch=master) for more information.  
> 
> If you need help getting set up to work in the private repo, see [How to get to the private repo](https://review.docs.microsoft.com/en-us/help/contribute/contribute-authors-in-public-repo?branch=master#how-to-get-to-the-private-repo). You can also take a look at the [GitHub account setup page](https://review.docs.microsoft.com/en-us/help/contribute/contribute-get-started-setup-github?branch=master) for additional details on making sure you're set up.

**Comment by KevinAsgari** [comment]
Michael Stahl is out, so I approved it as his manager.

**Comment by Jak-MS** [comment]
@FLY1NGSQU1RR3L - This URL isn't working for me: https://developer.microsoft.com/games/xbox/docs/gdk/provisioning-role ... can you check it? Thanks.

**Comment by KevinAsgari** [comment]
That URL links to restricted content which requires permission to view. We should probably clarify in the text that that page is only accessible to managed partners who have signed an NDA agreement.

File: windows-apps-src/xbox-apps/devkit-activation.md [hunk 1]
```diff
@@ -10,15 +10,20 @@ ms.localizationpriority: medium
...
 
-Developer Mode can be enabled on any retail Xbox One console. After Developer Mode is enabled, you can switch back and forth between Retail (**2a**) and Developer Modes (**2b**).
+Xbox retail consoles can have two modes, Retail Mode (1) and Developer Mode (2). In Retail Mode, the console is in its normal state: you can play games and run apps acquired through the Xbox store. In Developer Mode, you can develop and test software for the console, but you cannot play retail games or run retail apps.
+
+Developer Mode can be enabled on any retail Xbox console, via the “Retail to Dev Kit conversion” app found in the Xbox store. After Developer Mode is enabled on your retail console, you can switch back and forth between Retail (2a) and Developer Modes (2b).
+
+**NOTE: Do not run this conversion app on any Xbox development hardware acquired through an Xbox managed program** (e.g. ID@Xbox) or you may introduce errors and delays while developing your game. For information on activating development hardware see: https://developer.microsoft.com/en-us/games/xbox/docs/gdk/provisioning-role
+<br></br>
```
Comment (Jak-MS) [inline windows-apps-src/xbox-apps/devkit-activation.md]:
```suggestion
>[!NOTE]
>Do not run this conversion app on any Xbox development hardware acquired through an Xbox managed program (e.g. ID@Xbox) or you may introduce errors and delays while developing your game. If you're a managed partner, you can get more information on activating development hardware. Go to https://developer.microsoft.com/games/xbox/docs/gdk/provisioning-role  .
>

```

Comment (Jak-MS) [reply windows-apps-src/xbox-apps/devkit-activation.md]: If that looks good, we can merge these changes. @KevinAsgari

**Comment by KevinAsgari** [comment]
We're running this by our account managers to approve the wording, should have an answer for you shortly.

**Review by XboxNinge — APPROVED** [review]

**Comment by ktoliver** [comment]
#hold-off per @KevinAsgari

**Comment by KevinAsgari** [comment]
Wording is approved, go ahead and complete.

**Comment by PRMerger16** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
Thanks, all.

**Pull request merged**